ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/RecoveryTest.java

```diff
@@ -624,50 +624,6 @@ public void shouldNotDeleteTopicsOnRecoveryEvenIfLegacyDropCommandAlreadyInComma
     assertThat(topicClient.listTopicNames(), hasItem("B"));
   }
 
```

Comment:
Getting rid of these tests because they're invalid test scenarios now that servers can't enqueue conflicting commands to the command topic